model/riscv_vmem_types.sail

```diff
@@ -43,7 +43,7 @@ function accessType_to_str (a) =
     Read(_)            => "R",
     Write(_)           => "W",
     ReadWrite(_, _)    => "RW",
-    Execute()          => "X"
+    InstructionFetch() => "X"
```

Comment:
While we're changing this line can you add a trailing comma?

Reply:
Added.